source/content/glossar/bibliothek/dcat-examples/dataset-publisher-ttl.rst

```diff
@@ -1,5 +1,5 @@
 .. code-block:: Turtle
-    :caption: Publisher of a Dataset current implementation in Turtle: does not conform to DCAT-AP-CH
+    :caption: Publisher of a dataset: current implementation in Turtle
```

Comment:
Same as above: "Publisher of a dataset in Turtle"

Reply:
I agree, removed this too.